workshops/0_installation_party/README.md

```diff
@@ -17,54 +17,66 @@ Our goal is to get you set up with the base things you will need to participate
 
 ## Installing Python
 
-First!  
-Check if you already have python installed:
+First!
+Check if you already have python installed by writing the following in a terminal:
 
 ```sh
 python --version
 ```
 
-If you see the following output then you already have python installed and don’t need to do anything: 
+Linux and macOS might be familiar with terminals, if you are using windows you
```

Comment:
should this say "linux and macOS _users_ might...." or it means something else? In general great idea to add how to get the terminal up

Reply:
```suggestion
Linux and macOS users might be familiar with terminals, if you are using windows you
```